app/controllers/organizations/staff/adoption_application_reviews_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Organizations::Staff::AdoptionApplicationReviewsController < Organizations::BaseController
+  skip_before_action :authenticate_user!
```

Comment:
Why would we skip this on the staff scoped actions?

Reply:
Initially, my thought process was to comment out the `before_action :authenticate_user!` within the `Organizations::AdopterFosterer::AdopterApplicationsController` and `Organizations::AdopterFosterer::ProfilesController` and see tests fail. But all tests passed. So I moved the `before_action :authenticate_user!` to the `ApplicationController` and I got failing tests. Which made sense to me. I followed the errors and skipped authentication within those controllers. Eventually, all tests passed. Without skipping on the staff scoped actions, I get the following error
`Failure:
Organizations::Staff::StaffControllerTest#test_: authorization #activate should be authorized.  [test/controllers/organizations/staff/staff_controller_test.rb:25]:
Expected #<StaffAccount id: 2649, created_at: "2024-06-01 18:00:05.806002000 +0000", updated_at: "2024-06-01 18:00:05.806002000 +0000", organization_id: 5169, user_id: 4740, deactivated_at: nil> to be authorized with Organizations::StaffAccountPolicy#activate?, and context {}, but no such authorization has been made.
Registered authorizations: none`

Reply:
Hmm interesting. We can take a look in tomorrow meeting. My initial question is why would an authorizations related test be failing for an authentication action.

Reply:
That’s a good question I’ll also look into it more to see if I can do something differently

Reply:
Comment here if you find anything! We will discuss I’m on the weekly meeting tomorrow if you are interested in joining that. Ping me in slack for the link. It’s at 8 pm eastern time US.

Reply:
Will do! I sent a dm for the link